llama_index/vector_stores/types.py

```diff
@@ -158,15 +158,44 @@ def add(
         """Add embedding results to vector store."""
         ...
 
+    async def aadd(
```

Comment:
My brain is screaming looking at this function name LOL but I got nothing better to suggest

Reply:
cool so its not just me 😂 perhaps `async_add`?